sandbox.sh

```diff
@@ -516,6 +523,9 @@ deploy-dummy-ticket)
 deposit-dummy-ticket)
   deposit_ticket
   ;;
+load-test)
```

Comment:
@lykimq we can make a new PR that adds a `--added_latency` option that uses https://www.pico.net/kb/how-can-i-simulate-delayed-and-dropped-packets-in-linux/ to inject extra latency.